core/arch/arm/plat-aspeed/platform_ast2700.c

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, Aspeed Technology Inc.
+ */
+
+#include <console.h>
+#include <drivers/gic.h>
+#include <drivers/serial8250_uart.h>
+#include <mm/core_memprot.h>
+#include <platform_config.h>
+#include <stdint.h>
+#include <io.h>
+#include <kernel/boot.h>
+#include <kernel/panic.h>
```

Comment:
Sort in alphabetical order.
_kernel/panic.h_, _io.h_ and _stdint.h_ not needed.
Replace _mm/core_memprot.h_ with _mm/core_mmu.h_.